addons/mrp/models/mrp_production.py

```diff
@@ -177,10 +177,6 @@ def _get_default_location_dest_id(self):
     post_visible = fields.Boolean(
         'Allowed to Post Inventory', compute='_compute_post_visible',
         help='Technical field to check when we can post')
-    consumed_less_than_planned = fields.Boolean(
-        compute='_compute_consumed_less_than_planned',
-        help='Technical field used to see if we have to display a warning or not when confirming an order.')
-
```

Comment:
you should write in the commit message you're essentially reverting https://github.com/odoo/odoo/commit/d3617fd852dbe471ca667f8cf3573a9327d6b064